webpack/components/Table/EmptyStateMessage.js

```diff
@@ -1,11 +1,14 @@
+/* eslint-disable prefer-const */
```

Comment:
See the previous comment for a way to write this where you won't have to disable this rule. :)